src/diffusers/loaders/lora_pipeline.py

```diff
@@ -4249,7 +4249,32 @@ def lora_state_dict(
 
         return state_dict
 
-    # Copied from diffusers.loaders.lora_pipeline.CogVideoXLoraLoaderMixin.load_lora_weights
+    @classmethod
+    def _maybe_expand_t2v_lora_for_i2v(
+        cls,
+        transformer: torch.nn.Module,
+        state_dict,
+    ):
+        if any(k.startswith("blocks.") for k in state_dict):
+            num_blocks = len({k.split("blocks.")[1].split(".")[0] for k in state_dict})
+            is_i2v_lora = any("k_img" in k for k in state_dict) and any("v_img" in k for k in state_dict)
+            if not is_i2v_lora:
+                return state_dict
```

Comment:
We don't perform any extra operation if it's `is_i2v_lora`?

Reply:
This is for loading T2V lora into I2V model, so if it's already I2V lora we return the state dict as-is.